src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/Documents.java

```diff
@@ -26,7 +27,10 @@ private Documents() {
 
     private static Set<String> findDuplicates(List<Map<String, Object>> exceptionDocuments,
                                               List<Map<String, Object>> existingDocuments) {
-        return Sets.intersection(getDocumentIdSet(existingDocuments), getDocumentIdSet(exceptionDocuments));
+        return Sets.intersection(
+            getDocumentIdSet(existingDocuments),
+            getDocumentIdSet(exceptionDocuments)
+        );
```

Comment:
One way to ensure 'small PRs' mentioned during every retrospective is to refrain from completely unrelated changes.
Also git log/blame will make more sense...